contrib/fixup-libgfortran.sh

```diff
@@ -93,3 +93,12 @@ if [ "$UNAME" = "Darwin" ]; then
         done
     done
 fi
+
+if [ "$UNAME" = "Linux" ]; then
+    cd $private_libdir
+    for file in openlibm quadmath gfortran openblas arpack lapack openspecfun; do
+        for dylib in $(ls lib$file*.so* 2>/dev/null); do
+            patchelf --set-rpath \$ORIGIN $dylib
```

Comment:
That variable is only set in deps/, if you feel like rearranging things and threading it through, be my guest, but this code catches patchelf globally or in usr/tools, so I don't really see the need to do the extra work to make it work with an override `PATCHELF`.